apps/editing-toolkit/package.json

```diff
@@ -101,6 +101,7 @@
 		"@wordpress/base-styles": "2.0.1",
 		"@wordpress/block-editor": "*",
 		"@wordpress/blocks": "*",
+		"@wordpress/core-data": "^2.16.0",
```

Comment:
Hm, it's interesting how this was missing all the time. Guess it's a dependency of one of the other packages already.